web/src/views/Game/Points.tsx

```diff
@@ -157,9 +157,12 @@ export default function Points(props: {game: Game}) {
                 styles['color'] = 'grey'
               }
 
+              const prevEvent = idx > 0 ? arr[idx-1] : null;
+              const isCallahan = prevEvent ? ev.type === 'POINT' && prevEvent.type === "DEFENSE" && ev.firstActor === prevEvent.firstActor : false;
```

Comment:
could we change this to guard most of this logic to only run when the event is a point? That will save a bunch of lookups into the array that we don't need. It also lets us be a bit less defensive because we know there will be a previous event.

something like:
```js
let isCallahan = false
if (ev.type == 'POINT') {
  const prevEvent = arr[idx-1];
  isCallahan = prevEvent.type === "DEFENSE" && ev.firstActor === prevEvent.firstActor : false;
}
```

Reply:
sure, I can do that